### PR TITLE
Fix docs to say that the `Dockerfile` is created, not the image

### PR DESCRIPTION
**Pull request by sschuberth**
This is just a trivial docs fix without an associated issue.

### CODE REVIEW

File: src/docs/asciidoc/user-guide/23-tasks.adoc [hunk 1]
```diff
@@ -6,7 +6,7 @@ The plugin provides a set of tasks for your project and preconfigures them with
 |=======
 |Task name                 |Depends On                |Type                                                                                 |Description
 |`dockerSyncBuildContext`  |`classes`                 |TaskProvider<{uri-gradle-docs}/javadoc/org/gradle/api/tasks/Sync.html[Sync]>                       |Copies the application files to a temporary directory for image creation.
-|`dockerCreateDockerfile`  |`dockerSyncBuildContext`       |TaskProvider<{uri-ghpages}/api/com/bmuschko/gradle/docker/tasks/image/Dockerfile.html[Dockerfile]> |Creates the Docker image for the Java application.
+|`dockerCreateDockerfile`  |`dockerSyncBuildContext`       |TaskProvider<{uri-ghpages}/api/com/bmuschko/gradle/docker/tasks/image/Dockerfile.html[Dockerfile]> |Creates the `Dockerfile` for the Java application.
```
Comment (bmuschko) [inline src/docs/asciidoc/user-guide/23-tasks.adoc:9]: Can you make the same change in `33-tasks.adoc`?

**Review by bmuschko — CHANGES_REQUESTED** [review]
Great, thanks for this.

File: src/docs/asciidoc/user-guide/23-tasks.adoc (hunk 1 above)
Comment (sschuberth) [reply src/docs/asciidoc/user-guide/23-tasks.adoc:9]: Sure, commit amended and force-pushed.

**Comment by bmuschko** [comment]
Thanks for the pull request. It has been merged.

**Pull request merged**